.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Docker Publishing
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - '*'
+
+jobs:
+  docker-publish:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - run: git pull --unshallow
+    - name: Get Branch
+      uses: nelonoel/branch-name@v1
+    - name: Get Tag
+      uses: olegtarasov/get-tag@v2
+    - name: Set env vars for build
+      run: |
+        echo ::set-env name=BRANCH::$(echo ${BRANCH_NAME})
+        echo ::set-env name=TAG::$(echo ${GIT_TAG_NAME})
+    - name: Run commitish script
+      run: ./generate-commitish.sh
```

Comment:
Maybe a comment that this exports to `REACT_APP_VERSION`? Or better, is there a way to just set the env variable explicitly here? (I assume not, but if it's possible I think it would be better than a fixed side effect internal to the script.)

Reply:
Ya this is a tricky one.. There isn't a whole lot of support for collecting the output of a script inside a job.  I can change the name of the step to explicitly state what the `run` is doing if that helps.